src/language/printer.js

```diff
@@ -275,3 +275,17 @@ function printBlockString(value, isDescription) {
     ? `"""${escaped.replace(/"$/, '"\n')}"""`
     : `"""\n${isDescription ? escaped : indent(escaped)}\n"""`;
 }
+
+/**
+ * Prints variables one per line with a trailing comma, when there is more
+ * than one argument. Otherwise print inline.
+ */
+function printVariableDefinitions(args) {
+  if (!args || args.length === 0) {
+    return '';
+  }
+  if (args.length === 1) {
+    return '(' + args[0] + ')';
+  }
+  return '(\n' + indent(join(args, ',\n')) + '\n)';
```

Comment:
We were only using the comma due to having everything on one line. A newline is a perfectly acceptable variable definition separator, just as it is for fields inside a document.

Reply:
TIL!